### PR TITLE
AT 11.03.15 | New view > Edit View > Change the Name on the Existing View Using an Illegal Character from the List of Illegal Characters

### PR DESCRIPTION
**Pull request by SergeDot**
AT 11.03.15 | New view > Edit View > Change the Name on the Existing View Using an Illegal Character from the List of Illegal Characters
Pre-conditions:
Login Jenkins, create a view, start from Dashboard Page.

Steps:

    Click on My Views
    Click on a View
    Click Edit View menu option
    Change the view name using an illegal character form the predefined list {"#!@$%^&*:;<>?/[]|\"}
    Click the OK button
    Repeat the steps for each character in the list until teaching the end of the list

**Expected result: **
There is an error page informing that an illegal character is an unsafe character for each of the characters from the list. A new view is not created

### CODE REVIEW

**Comment by SergeDot** [comment]
TC 11.03.15 | New view > Edit View > Change the Name on the Existing View Using an Illegal Character from the List of Illegal Characters

Pre-conditions:
Login Jenkins, create a view, start from Dashboard Page.

Steps:

    Click on My Views
    Click on a View
    Click Edit View menu option
    Change the view name using an illegal character form the predefined list {"#!@$%^&*:;<>?/[]|\"}
    Click the OK button
    Repeat the steps for each character in the list until teaching the end of the list

**Expected result: **
There is an error page informing that an illegal character is an unsafe character for each of the characters from the list. A new view is not created

**Comment by SergeDot** [comment]
[AT 11.03.15 | New view > Edit View > Change the Name on the Existing View Using an Illegal Character from the List of Illegal Characters](https://trello.com/c/01CASY13/706-at-110315-new-view-edit-view-change-the-name-on-the-existing-view-using-an-illegal-character-from-the-list-of-illegal-characters)

File: src/test/java/EditViewTest.java [hunk 1]
```diff
@@ -296,8 +300,40 @@ public void testDeleteColumn() {
...
+                        .noneMatch(element -> element.getText()
+                                .equals(String.format("‘%c’ is an unsafe character", illegalCharacters[finalI]))));
+                goToEditView();
+            } else {
+                checks.add(false);
+            }
+        }
+
+        Assert.assertTrue(checks.stream().allMatch(element->element == true));
+        deleteAllViews();
```
Comment (snysmymric) [inline src/test/java/EditViewTest.java:335]: А можно тут использовать метод Сергея deleteViews()  из JenkinsUtils ?

**Review by snysmymric — APPROVED** [review]
LGTM

File: src/test/java/EditViewTest.java (hunk 1 above)
Comment (SergeDot) [reply src/test/java/EditViewTest.java:335]: Я хочу довыложить еше 2 теста и сделать больгой рефакторинг интегрируя новые методы Сергея плюс в целом этот код надо дооптимизировать так как сейчас там стоят костыли для фикса падений когда они все вместе запускаются. Как только остальные 2 теста и этот будут одобрены, будет проведена большая работа.

**Review by UmidaKaharova — APPROVED** [review]

**Pull request merged**